centreon/www/install/php/Update-24.04.0.php

```diff
@@ -211,6 +211,25 @@
     }
 };
 
+$populateDahsboardTables = function(CentreonDb $pearDB): void {
```

Comment:
Unnecessary update file.
If the user comes from a version lower than 23.02, the update-23.10.2 file contains all the code needed to update the database.
From a new installation, the modification will be made with the inserBaseConf.sql file.